.travis.yml

```diff
@@ -9,6 +9,7 @@ node_js:
   - "0.10"
 env:
   - TEST_DIR=packages/fluxible
+  - TEST_DIR=packages/fluxible-reducer-store
```

Comment:
I had to update the scripts in package.json for each package to point to the root binaries. It's now working as I mentioned above now.